airbyte-integrations/connectors/source-newsdata/source_newsdata/newsdata.yaml

```diff
@@ -0,0 +1,66 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["results"]
+  base_requester:
+    url_base: "https://newsdata.io/api/1"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "X-ACCESS-KEY"
+      api_token: "{{ config['api_key'] }}"
+  base_retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
```

Comment:
@Xabilahu can you check the pagination for this connector?
<img width="306" alt="image" src="https://user-images.githubusercontent.com/5154322/198569783-3886349e-a690-45af-8b42-d8607fc6aa10.png">